YashasSingh.md

```diff
@@ -0,0 +1,65 @@
+---
+name: "@YashasSingh"
+project: "RPI Ad blocker, Virtual Assitant Hologram Alarm Clock, "
+---
+
+# RPI ad blocker
+
+## Summary
+Internet ads are always annoying so why not create a network wide ad blocker. This will allow me moniter the Wifi as well as block ads.
+## Plan
+
+Use pi hole to create on the Raspberry pi zero 
+Install Pi hole 
+connect the rasberry pi to the wifi so it blocks all ads on the network
+
+## Budget
+
+What materials will you need for your project? Where will you get them? How much does it cost? Please include all materials, including components you already own. Make sure to factor in shipping costs and sales tax.
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Rasberry Pi 0 w | (https://www.amazon.com/gp/product/B07CMVDHWB/ref=ewc_pr_img_4?smid=A30ZYR2W3VAJ0A&psc=1)| $79.99 |
+| Wifi router/moderm| Already own|$0.00|
+| Amazon shipping| none | free|
+  
+|Total| $79.99|
+  
+  
+ # Virtual Assitant Hologram Alarm Clock
+ 
+ 
+ ## Summary 
+ 
+ Virtual Assitant Hologram Alarm Clock or, VAHAC for short, is a virtual assitant just like Alexa or Google Home. This also funtions as a hologram Alarm clock.
+ The rasberrypi will use a camera to detect facial expressions and faces using open-cv in order to suggest information or help. 
+ For example if the user gives a face of pain then the assitant can tell the user what do. 
+ The hologram feature will show a random pictures like jellyfish floating when the virtual assitant is not in use.
+ This project also features an alarm clock which will play music until the user is awake and out of the room.
+ 
+ 
+ ## Plan
+ 
+ Set up rasberry pi
+ Connect the screen, camera, audio hat module
+ Install a voice reconigition software to convert wavelengths into text
```

Comment:
Can you link an example?

Reply:
Can you add this to your markdown document?

Reply:
Okay sure